packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -213,7 +213,7 @@ async function getOrderNumberFromSession(
 
   if (cookieSession) {
     const { namespaces } = await commerce.getSessionOrder()
-    return namespaces.checkout?.orderFormId?.value
+    return namespaces.checkout?.orderFormId?.value ?? undefined
```

Comment:
if it has no value, it will return undefined, right?

Reply:
yes @emersonlaurentino !

So the orderNumberFromSession it would return undefined and the orderNumber would be the orderNumberFromCart (local storage)